cli/src/commands/branch.rs

```diff
@@ -98,7 +98,7 @@ pub struct BranchDeleteArgs {
 pub struct BranchListArgs {
     /// Show all tracking and non-tracking remote branches including the ones
     /// whose targets are synchronized with the local branches.
-    #[arg(long, short, conflicts_with_all = ["names", "revisions", "tracked"])]
+    #[arg(long, short, conflicts_with_all = ["names", "tracked"])]
```

Comment:
> Also discussed was renaming `--all` to `--all-remotes`, but this change is a bit longer and will be followed in a separate PR.

Perhaps, `--all` can be kept as a hidden alias (if you mean it's not simple to update all tests and add deprecation handling.)

Reply:
It's not too difficult, just a bit noisy PR. I can add an alias, sure